### PR TITLE
Hide secrets

### PR DESCRIPTION
**Pull request by sushmita56**
I've hidden API key and Django secret key inside environment variables and then accessed that information from the code for the security purpose.  These changes allow me to conceal my secret keys in the windows environment variable and read them in code without actually writing them directly in the code.

### CODE REVIEW

File: weather/weatherapp/views.py [hunk 1]
```diff
@@ -3,17 +3,52 @@
...
+
+        if error_msg:
+            message = error_msg
+            message_class = 'is-danger'
+        else:
+            message = 'City added successfully'
+            message_class= 'is-success'
+
+
+    # if request.method == 'POST':
```
Comment (ritushishir) [inline weather/weatherapp/views.py:45]: can be removed this section?

File: weather/weatherapp/views.py [hunk 2]
```diff
@@ -3,17 +3,52 @@
...
+            existing_city_count = City.objects.filter(name=new_city).count()
 
-    cities = City.objects.all()
+            if existing_city_count == 0:
+                r = requests.get(url.format(new_city)).json()
+                
+                if r['cod'] == 200:
+                    form.save()
+                else:
+                    error_msg = 'City doesnot exists.'
```
Comment (ritushishir) [inline weather/weatherapp/views.py:33]: "City does not exist"?

File: weather/weatherapp/views.py [hunk 3]
```diff
@@ -3,17 +3,52 @@
...
 from .models import City
 from .forms import CityForm
+import os 
+
+
+apiId = os.environ['weatherapp_api_key']
 
 def index(request):
-    url = 'http://api.openweathermap.org/data/2.5/weather?q={}&units=imperial&appid= '
+    url = 'http://api.openweathermap.org/data/2.5/weather?q={}&units=imperial&appid=%s' %apiId
```
Comment (ritushishir) [inline weather/weatherapp/views.py:12]: you are adding **apiID** at the end of the URL string, right? 
not sure about the syntax but it looks like something is missing. Does it work?

**Review by ritushishir — CHANGES_REQUESTED** [review]

**Comment by ritushishir** [comment]
Seems a lot of changes but it's great that the secrets are hidden now.  @sushmita56 👍

File: weather/weatherapp/views.py (hunk 3 above)
Comment (sushmita56) [reply weather/weatherapp/views.py:12]: Yes! It does. I had run the code before committing the changes. It works well.

File: weather/weatherapp/views.py (hunk 1 above)
Comment (sushmita56) [reply weather/weatherapp/views.py:45]: No, this part of the code is responsible for letting us know whether the "city" already exists  in the database or not. So I think it's good to be.

**Review by ritushishir — APPROVED** [review]

**Review by ritushishir — APPROVED** [review]

**Pull request merged**